docs/user/development.rst

```diff
@@ -6,30 +6,32 @@ GitHub. The development began in September 2014.
 
 It is developed on GitHub: https://github.com/Lasagne/Lasagne
 
-You can file issues and feature requests there.
+You can file feature requests and bug reports there. If you have questions
+about how to use the library, please post on the lasagne-users mailing list
+instead: https://groups.google.com/forum/#!forum/lasagne-users
 
 Contributions
 -------------
 
-Everybody is welcome to contribute to ``Lasagne``. You can do so by
+Everybody is welcome to contribute to Lasagne. You can do so by
 
-* Testing it and giving us feedback / opening issues on GitHub.
+* Testing it and giving us feedback / submitting bug reports on GitHub.
 
-  * Writing unittests.
-
-  * Simply using the software.
-
-* Writing new code and sending it to us as pull requests. However, before you
-  add new functionality you should eventually ask if we really want that as
-  part of our project.
+* Writing unit tests.
 
 * Improving existing code.
 
-* Suggesting something else how you can contribute.
+* Writing new code and sending pull requests on GitHub. Note that Lasagne
+  has a fairly narrow focus and we strictly follow a set of design principles
+  (see below), so we cannot guarantee upfront that your pull request will
+  be accepted.
```

Comment:
To make it a bit friendlier, possibly add something like: "You're of course more than welcome to discuss your idea in a github issue or on the mailing list first before sending in a pull request."